common/src/main/java/com/wynntils/features/ui/WynncraftButtonFeature.java

```diff
@@ -90,7 +90,7 @@ private ServerData getWynncraftServer() {
 
     private static void connectToServer(ServerData serverData) {
         ConnectScreen.startConnecting(
-                McUtils.mc().screen, McUtils.mc(), ServerAddress.parseString(serverData.ip), serverData);
+                McUtils.mc().screen, McUtils.mc(), ServerAddress.parseString(serverData.ip), serverData, false);
```

Comment:
Just curious: what does the false represent?

Reply:
`isQuickPlay` or smth like that. You know, you can have a config file to auto join a server on launch.

Reply:
Ah, thanks.